pynicotine/gtkgui/ui/search.ui

```diff
@@ -6,7 +6,7 @@
     <property name="max-width-chars">0</property>
     <property name="placeholder-text" translatable="yes">Include text…</property>
     <property name="primary-icon-name">emblem-ok-symbolic</property>
-    <property name="tooltip-text" translatable="yes">Filter in results whose file paths contain the specified text. Multiple phrases and words can be specified, e.g. exact phrase|music|term|exact phrase two</property>
+    <property name="tooltip-text" translatable="yes">Add a filter to only show results whose file paths have the given text. Use a '|' to specify multiple words or phrases, for example: stairway to|misty mountain</property>
```

Comment:
The codebase must not contain any phrases that reference copyrighted materials.

Reply:
Right. Let me think...